platform/view/services/comm/host/rest/websocket/multiplexed_provider.go

```diff
@@ -63,21 +77,38 @@ func NewMultiplexedProvider(tracerProvider trace.TracerProvider, metricsProvider
 		})),
 		m: newMetrics(metricsProvider),
 	}
+
+	// spawn ActiveSubConn tracker
+	p.trackerDone = StartTracker(func() {
+		sum := 0
+		p.mu.RLock()
+		for _, c := range p.clients {
+			c.mu.RLock()
+			sum += len(c.subConns)
```

Comment:
I would avoid the locks here. You only need eventually consistent results, so I think it would be better to keep an atomic.Int for each client that keeps the length of the subConns. Then here you sum them. These results are just for tracking purposes (or for sanity checks in our tests), so using locks in production may add unnecessary overhead.

Reply:
I did not touch the existing logic of the metric tracker; just moved it in order to be able to cancel the go routine - if needed. I would improve the actual tracking in another PR